boards/arm/nrf5340_dk_nrf5340/nrf5340_cpunet_reset.c

```diff
@@ -39,6 +39,11 @@ static void remoteproc_mgr_config(void)
 	 * accesses by Network MCU will have Secure attribute set).
 	 */
 	NRF_SPU->EXTDOMAIN[0].PERM = 1 << 4;
+
+	/* Treat shared memory as BSS section and initialize it to zero. */
+	BUILD_ASSERT_MSG((void *) DT_IPC_SHM_BASE_ADDRESS != NULL,
```

Comment:
Instead of a build-time assert, I wonder if you can just `#ifd DT_IPC_SHM_BASE_ADDRESS != 0 ` before calling memset
Not sure if the build-time assert makes sense to be default, as shared memory is not required in default nRF5340 support

Reply:
fixed